manifests/g/gileli121/windowtop/5.22.5/gileli121.windowtop.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with WinGet Automation using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: gileli121.windowtop
+PackageVersion: 5.22.5
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-11-16
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
I believe the above is an architecture error.